seacrowd/sea_datasets/vlsp2020_mt_envi/vlsp2020_mt_envi.py

```diff
@@ -0,0 +1,162 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Parallel and monolingual data for training machine translation systems translating English texts into Vietnamese, with a focus on news domain. The data was crawled from high-quality bilingual or multilingual websites of news and one-speaker educational talks on various topics, mostly technology, entertainment, and design (hereby referred to as TED-like talks). The dataset also includes noisy movie subtitles from the OpenSubtitle dataset.
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{vlsp2020-mt,
+title     = {{Goals, Challenges and Findings of the VLSP 2020 English-Vietnamese News Translation Shared Task}},
+author    = {Thanh-Le Ha and Van-Khanh Tran and Kim-Anh Nguyen},
+booktitle = {{Proceedings of the 7th International Workshop on Vietnamese Language and Speech Processing - VLSP 2020}},
+year      = {2020}
+}
+"""
+
+_DATASETNAME = "vlsp2020_mt_envi"
+
+_DESCRIPTION = """\
+Parallel and monolingual data for training machine translation systems translating English texts into Vietnamese, with a focus on news domain. The data was crawled from high-quality bilingual or multilingual websites of news and one-speaker educational talks on various topics, mostly technology, entertainment, and design (hereby referred to as TED-like talks). The dataset also includes noisy movie subtitles from the OpenSubtitle dataset.
+"""
+
+_HOMEPAGE = "https://github.com/thanhleha-kit/EnViCorpora"
+
+_LANGUAGES = ["vie"] 
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = "https://github.com/thanhleha-kit/EnViCorpora/archive/refs/heads/master.zip"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+class Vlsp2020MtEnviDataset(datasets.GeneratorBasedBuilder):
+    """Parallel and monolingual data for training machine translation systems translating English texts into Vietnamese, with a focus on news domain. The data was crawled from high-quality bilingual or multilingual websites of news and one-speaker educational talks on various topics, mostly technology, entertainment, and design (hereby referred to as TED-like talks). The dataset also includes noisy movie subtitles from the OpenSubtitle dataset."""
+
+    # Skipping openSub & mono-vi for future development (Large Drive file download bottleneck)
+    subsets = {
+        # key: subset_id, value: subset_filename
+        "EVBCorpus" : ["bitext"],
+        "VLSP20-official" : ["offi_test"],
+        "basic": ["data"],
```

Comment:
quick qn:
for these three, does the author/experiment note these belong to the `train` subset?

Reply:
Hi @sabilmakbar, sorry for the late reply ! 

There is no clear separation on the single-split subsets, hence I implemented it to the `train` subset first 🙏

Reply:
hmm, after I checked their README, it's clearly stated that some are test-only data
https://github.com/thanhleha-kit/EnViCorpora?tab=readme-ov-file#statistics-of-developmenttest-data

do you mind to revisit each of the folders and follow the train-dev-test splitting based on the available foldername there? If no info is given on filename & README, we can assume it as `train` for the last resort